src/CoreWCF.Primitives/src/CoreWCF/Channels/SecurityBindingElement.cs

```diff
@@ -841,6 +885,51 @@ static public SecurityBindingElement CreateSecureConversationBindingElement(Secu
             }
             return result;
         }
+
+        // If any changes are made to this method, please make sure that they are
+        // reflected in the corresponding IsSspiNegotiationOverTransportBinding() method.
+        static public TransportSecurityBindingElement CreateSspiNegotiationOverTransportBindingElement(bool requireCancellation)
+        {
+            TransportSecurityBindingElement result = new TransportSecurityBindingElement();
+            SspiSecurityTokenParameters sspiParameters = new SspiSecurityTokenParameters(requireCancellation);
+            sspiParameters.RequireDerivedKeys = false;
+            result.EndpointSupportingTokenParameters.Endorsing.Add(
+                sspiParameters);
+            result.IncludeTimestamp = true;
+           // result.LocalClientSettings.DetectReplays = false;
+            result.LocalServiceSettings.DetectReplays = false;
+            result.SupportsExtendedProtectionPolicy = true;
+
+            return result;
+        }
+
+        // this method reverses CreateSspiNegotiationOverTransportBindingElement() logic
+        internal static bool IsSspiNegotiationOverTransportBinding(SecurityBindingElement sbe, bool requireCancellation)
+        {
```

Comment:
This method isn't needed. It's used when parsing config

Reply:
done